cdhweb/pages/management/commands/exodus.py

```diff
@@ -105,6 +105,10 @@ def handle(self, *args, **options):
         site.save()
         Page.objects.filter(depth=2).exclude(pk=homepage.pk).delete()
 
+        # set the site's hostname so that sitemaps will be valid
+        site.hostname = "cdh.princeton.edu"
```

Comment:
do we need to do anything for qa, or does it not matter?

Reply:
shouldn't matter since QA can't be crawled without VPN, i think